app/processing/invoice-line/invoice-line-schema.js

```diff
@@ -0,0 +1,5 @@
+const Joi = require('joi')
+
+module.exports = Joi.object({
+  value: Joi.number().integer().required()

```

Comment:
Needs to be greater than 0 -- as per the db select

Reply:
May be zero, hence added min(0) validation